Buildings/Experimental/DHC/CentralPlants/Cooling/Subsystems/Examples/CoolingTowerParallel.mo

```diff
@@ -0,0 +1,171 @@
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    dp_nominal=60000,
+    TAirInWB_nominal=273.15+25.55,
+    TWatIn_nominal=273.15+35,
+    dT_nominal=5.56,
+    PFan_nominal=4800)
+    "Parallel cooling towers"
+    annotation (Placement(transformation(extent={{0,40},{20,60}})));
+  Buildings.BoundaryConditions.WeatherData.ReaderTMY3 weaDat(
+    final computeWetBulbTemperature=true,
+    filNam=Modelica.Utilities.Files.loadResource(
+      "modelica://Buildings/Resources/weatherdata/USA_CA_San.Francisco.Intl.AP.724940_TMY3.mos"))
+    annotation (Placement(transformation(extent={{-100,80},{-80,100}})));
+  Modelica.Blocks.Logical.OnOffController onOffCon(
+    bandwidth=2,
+    reference(
+      unit="K",
+      displayUnit="degC"),
+    u(
+      unit="K",
+      displayUnit="degC"))
+    "On/off controller"
+    annotation (Placement(transformation(extent={{-10,-90},{10,-70}})));
+  Modelica.Blocks.Logical.Switch swi
+    "Control switch for chilled water pump"
+    annotation (Placement(transformation(extent={{30,-90},{50,-70}})));
+  Modelica.Blocks.Sources.Constant TSwi(
+    k=273.15+22)
+    "Switch temperature for switching tower pump on"
+    annotation (Placement(transformation(extent={{-60,-96},{-40,-76}})));
+  Modelica.Blocks.Sources.Constant zer(
+    k=0)
+    "Zero flow rate"
+    annotation (Placement(transformation(extent={{-10,-120},{10,-100}})));
+  Modelica.Blocks.Sources.Constant m_flow(
+    k=m_flow_nominal)
+    "Water flow rate"
+    annotation (Placement(transformation(extent={{-10,-50},{10,-30}})));
+  Buildings.Fluid.MixingVolumes.MixingVolume vol(
+    nPorts=3,
+    redeclare package Medium=Medium,
+    m_flow_nominal=m_flow_nominal,
+    V=0.5,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial)
+    annotation (Placement(transformation(extent={{20,-20},{40,0}})));
+  Buildings.Fluid.Sources.Boundary_pT exp(
+    redeclare package Medium=Medium,
+    nPorts=1)
+    "Expansion vessel"
+    annotation (Placement(transformation(extent={{100,-30},{80,-10}})));
+  Modelica.Thermal.HeatTransfer.Sources.FixedHeatFlow fixHeaFlo(
+    Q_flow=0.5*m_flow_nominal*4200*5)
+    "Fixed heat flow rate"
+    annotation (Placement(transformation(extent={{-40,0},{-20,20}})));
+  Modelica.Thermal.HeatTransfer.Sensors.TemperatureSensor TVol
+    "Water temperature"
+    annotation (Placement(transformation(extent={{-60,-50},{-40,-30}})));
+  Buildings.Controls.Continuous.LimPID conFan(
+    k=1,
+    Ti=60,
+    Td=10,
+    reverseActing=false,
+    u_s(
+      unit="K",
+      displayUnit="degC"),
+    u_m(
+      unit="K",
+      displayUnit="degC"))
+    "Controller for tower fan"
+    annotation (Placement(transformation(extent={{-20,80},{0,100}})));
+  Modelica.Blocks.Sources.Constant TSetLea(
+    k=273.15+18)
+    "Setpoint for leaving temperature"
+    annotation (Placement(transformation(extent={{-60,80},{-40,100}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTCWLvg(
```

Comment:
The circled real output in `cooTowPar` below is the vector `TLvg` from each individual cooling tower. With the parallel cooling tower model, I wonder if the mixed leaving temperature would be more useful? If it is the single mixed leaving temperature instead, then `senTCWLvg` is not needed. I'd be curious to hear your thoughts. 
(Eliminating `senTCHLvg` from this example model would also allow you to extend `PartialStaticTwoPortCoolingTower`, I believe.)
![image](https://user-images.githubusercontent.com/44682485/118852539-1dd17600-b890-11eb-8e54-cd4e06f5fec2.png)